### ISSUE TITLE
Pressure sensitivity doesn't work (without TabletPC)

### ISSUE THREAD

**Issue by ReturnMinusOne** [opentoonz/opentoonz#3865]
**Describe the bug**

Pressure sensitivity no longer seems to work in OpenToonz 1.5 RC 2 without enabling TabletPC in my tablet driver.

This is with a Huion WH1409 drawing tablet (original model with 2048 levels of pressure).

**To Reproduce**

Launch OpenToonz and start drawing, without TabletPC enabled by the tablet driver. I imagine this problem does not apply to all drawing tablets.

**Expected behavior**

OpenToonz 1.4 has pressure sensitivity working with or without TabletPC; it seems that version 1.5 has regressed unexpectedly in this aspect.

**Device Information:**
 - OS: Windows 10 (64-bit)
 - OpenToonz Version: 1.5 Release Candidate 2
 - GPU: AMD (Gigabyte) Radeon RX 590
 - GPU Driver Version: 20.10.27.05-201027a-360165C-RadeonProEnterprise

**Comment by shun-iwasawa** [comment]
In OT V1.5 we updated the underlying library (Qt) from version 5.9 to 5.15 .
One of the major changes between two versions is migrating fundamental tablet API from WinTab to Windows Ink .
Since Qt's native Windows Ink support behaved badly with our software, we decided to add a modification to re-introduce WinTab support to the library.

Currently OT V1.5 is supposed to be available with both types of Tablet API as well as V1.4. However, since the underlying library is altered, some additional configuration may be needed as in the case of yours.
Sorry for the inconvenience.

**Comment by ReturnMinusOne** [comment]
Understood. Really my main problem with activating TabletPC is that clicking on text fields with the stylus displays a very irritating Windows Ink prompt. Apparently Windows Ink can be disabled through the registry, so that may be my best option.

Thank you.

**Comment by oliver-dreamwalker** [comment]
I have a similar problem, but I when I tested this PR  #3823 the pressure sensitivity worked fine.

**Comment by ReturnMinusOne** [comment]
The latest distribution of Tahoma 2D uses Qt 5.9.9, which I think is also the version used by OpenToonz 1.4; since both programs have functional stylus pressure, it's very likely that this is indeed an issue with the new version of Qt. However, this doesn't appear in the [list of known issues for Qt 5.15.2](https://wiki.qt.io/Qt_5.15.2_Known_Issues) (used by OT 1.5).

Hopefully, compatibility will improve over time. But if not, enabling TabletPC isn't disastrous for me, just inconvenient.

**Comment by shun-iwasawa** [comment]
@ReturnMinusOne 
We noticed that the library included in V1.5RC2 was not proper. I just replaced the installer which may work as you expected.
I'm so sorry for bothering you, but could you please install https://github.com/opentoonz/opentoonz/releases/download/v1.5.0rc2/OpenToonzSetup.exe and try the tablet behavior again? Thank you!

**Comment by ReturnMinusOne** [comment]
Done, and it works!

I completely uninstalled the program, deleted the stuff folder and rebooted to make sure the install would be clean.

After installation, I tested it with default settings (and without TabletPC), and pressure sensitivity worked as expected.

I tried turning on TabletPC to see what would happen. The Windows Ink prompt seems to show up, sporadically, when I return the focus to the window, then click a text field, but doesn't reappear after that. Regardless, it's no longer necessary.

Thank you for the update!
